Makefile

```diff
@@ -154,3 +154,6 @@ mock:
 	mockgen -source pkg/p2p/peer/peer.go -destination pkg/mock/peer.go -package mock Peer
 	mockgen -source pkg/state/api.go -destination pkg/mock/state.go -package mock State
 	mockgen -source pkg/node/state_fsm/default.go -destination pkg/node/state_fsm/default_mock.go -package state_fsm Default
+
+proto:
+	./genpb.sh
```

Comment:
Let's move commands for generation here and remove `genpb.sh`. It will be the same as for mocks generation. Later we can introduce a `generate` command that do both.